src/main/java/uk/ac/ebi/ena/webin/cli/WebinCli.java

```diff
@@ -232,13 +232,21 @@ private static WebinCliParameters initParameters(WebinCliCommand cmd) {
                     .setPassword(parameters.getPassword())
                     .setTest(parameters.isTest())
                     .build();
-            submitService.doSubmission(bundle.getXMLFileList(), bundle.getCenterName(), getVersionForSubmission());
+            submitService.doSubmission(bundle.getXMLFileList(), bundle.getCenterName(), getVersionForSubmission(), bundle.getManifestMd5(), getManifestFileContent());
 
         } catch (WebinCliException e) {
             throw WebinCliException.error(e, WebinCliMessage.CLI_SUBMIT_ERROR.format(e.getErrorType().text));
         }
     }
 
+    String getManifestFileContent() {
+        String manifestFileContent = "  <![CDATA[ \n%s\n ]] >";
```

Comment:
Would it not be better to pass in as a string, and when injecting into XML, then wrap into the CDATA block?